src/sidebar/store/modules/toast-messages.js

```diff
@@ -6,6 +6,7 @@ import * as util from '../util';
  * @prop {string} id
  * @prop {string} message
  * @prop {string} moreInfoURL
+ * @prop {boolean} isDismissed
```

Comment:
this appeared to be missing and I don't know if we intended this to be optional, but I'd like to error on the side of strict.